.github/workflows/test-template.yml

```diff
@@ -207,14 +217,21 @@ jobs:
           # Fails if set to true
           skip-tsc: false
 
-      - run: pnpm run test:functional --data-proxy --shard ${{ matrix.shard }}
+      - name: Get Preview Features
+        id: gpf
+        run: ./.github/workflows/scripts/get-preview-features.js
+        env:
+          PREVIEW_FEATURE_SET: ${{ matrix.previewFeatureSet }}
+          EXCLUDED_PREVIEW_FEATURES: driverAdapters,metrics
```

Comment:
We don't enable these for the Data Proxy tests. Maybe driver adapters could make sense though?

Reply:
```suggestion
          # We skip `metrics` for Data Proxy tests
          EXCLUDED_PREVIEW_FEATURES: driverAdapters,metrics
```

Reply:
I'm not sure for `driverAdapters` here if that actually changes something? 🤔

Reply:
Yes, that's a good question. I'll give it some thought